.github/workflows/test.yml

```
@@ -123,3 +129,18 @@ jobs:
         with:
           name: server-logs
           path: server.log
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)